<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal4</class>
 <widget class="QWidget" name="Principal4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Principal4{
	background:#212121;
}

.QLabel{
	background:#212121;
}

.QLineEdit{
	
	font: 75 7.5pt &quot;Arial&quot;;
}
.QPushButton{
	font-family: Arial, Verdana;
	color:#ffffff;
	background-color: #1883BA;
	padding:5px;
	font-weight: 600;
	font-size: 13px;
}
</string>
  </property>
  <widget class="QLineEdit" name="txtLanda">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLineaEspera">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Simular</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNiu">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>N째.Eventos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tasa de llegada</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEventosLinea">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla_linea_espera">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>1001</width>
     <height>281</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>9</number>
   </property>
   <column>
    <property name="text">
     <string>A. Llegada</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>A. Servicio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tiempo entre llegada</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tiempo de servicio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hora de llegada</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hora iniciaci처n de servicio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hora terminaci처n de servicio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tiempo de Espera</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tiempo en el Sistema</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulaci처n Linea de Espera</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tasa de servicio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnObtenerValores">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Obtener valores</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotales">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Totales (min):</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPromedios">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Promedios (min):</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalTiempoEntreLlegada">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="totalTiempoServicio">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="promedioTiempoEntreLlegada">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="promedioTiempoServicio">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="totalTiempoEspera">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="totalTiempoSistema">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="promedioTiempoEspera">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="promedioTiempoSistema">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtLanda</tabstop>
  <tabstop>txtNiu</tabstop>
  <tabstop>txtEventosLinea</tabstop>
  <tabstop>btnLineaEspera</tabstop>
  <tabstop>btnObtenerValores</tabstop>
  <tabstop>tabla_linea_espera</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
